docs/reference/release-notes/7.8.asciidoc

```diff
@@ -173,6 +173,37 @@ Machine Learning::
 * Skip daily maintenance activity if upgrade mode is enabled {pull}54565[#54565] (issue: {issue}54326[#54326])
 * Start gathering and storing inference stats {pull}53429[#53429]
 * Unassign DFA tasks in SetUpgradeModeAction {pull}54523[#54523] (issue: {issue}54326[#54326])
```

Comment:
@lcawl what is the official docs position on using the DFA acronym in public documentation? If it isn't supposed to be used then it should be expanded to "data frame analytics" here.